libs/ragulate/ragstack_ragulate/datasets/utils.py

```diff
@@ -1,17 +1,19 @@
 import os
+from os.path import dirname, abspath
 
 import inflection
 
 from .base_dataset import BaseDataset
 from .crag_dataset import CragDataset
 from .llama_dataset import LlamaDataset
 
+DATASET_ROOT_PATH = dirname(abspath(__file__))
```

Comment:
this is incorrect. it is not the path of this code folder.  it is a folder in the user's directory to store datasets.

I'll revert this change and make a fix for when it doesn't exist.